handler/SelfHandler.py

```diff
@@ -2,6 +2,7 @@
 
 import socket
 from utils import net_utils
+from utils import shell_colors
```

Comment:
Piccolezza, in Fapster io la importavo con un ```as shell``` così nel codice si leggeva ```shell.print_blue()```. Vedi tu se hai cazzi o non te ne frega una sega 😆